build.php

```diff
@@ -148,15 +148,31 @@
 };
 
 $buildDesignStructure = function ($vendor, $nameSpace) {
-    $layoutFolder   = 'app/design/frontend/base/default/layout/' . strtolower($vendor);
-    $templateFolder = 'app/design/frontend/base/default/template/' . strtolower($vendor) . '/' . strtolower($nameSpace);
-    $skinJsFolder   = 'skin/frontend/base/default/js/' . strtolower($vendor) . '/' . strtolower($nameSpace);
 
-    mkdir($layoutFolder, 0777, true);
-    mkdir($templateFolder, 0777, true);
+    $layoutDirs = [
+        "frontend" => 'app/design/frontend/base/default/layout/' . strtolower($vendor),
+        "adminhtml" => 'app/design/adminhtml/base/default/layout/' . strtolower($vendor)
+    ];
+
+    $templateDirs = [
+        "frontend" => 'app/design/frontend/base/default/template/' . strtolower($vendor) . '/' . strtolower($nameSpace),
+        "adminhtml" => 'app/design/adminhtml/base/default/template/' . strtolower($vendor) . '/' . strtolower($nameSpace),
+    ];
+
+    foreach ($layoutDirs as $layoutDir) {
+        mkdir($layoutDir, 0777, true);
+        unlink('app/design/frontend/base/default/layout/.gitkeep');
```

Comment:
These unlinks need to be outside the loop as the second call emits a warning as it no longer exists